src/main/java/duke/command/DeadlineCommand.java

```diff
@@ -0,0 +1,50 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.task.Deadline;
+import duke.task.TaskList;
+import duke.ui.Ui;
+import duke.util.DateTimeUtils;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+
+public class DeadlineCommand extends AddTaskCommand {
```

Comment:
Great use of OOP here! Similar for DeleteCommand, EventCommand, and ToDoCommand.